src/task_monitor/tasks/process_identities.rs

```diff
@@ -477,7 +477,7 @@ pub async fn delete_identities(
 
     // Next get merkle proofs for each update - note the proofs are acquired from
     // intermediate versions of the tree
-    let mut merkle_proofs: Vec<_> = updates
+    let merkle_proofs: Vec<_> = updates
```

Comment:
Don't we need to pad out merkle proofs with essentially all zero bytes?

I know we say that prover will skip the proofs but afaik it's a kind of a branchless programming situation and you do actually this data to be there